android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -106,7 +106,6 @@ public void run() {
     public void run() {
       Log.d(TAG, "running second scan timeout");
       emptyGroupIdQueue();
-      showListView();
```

Comment:
Correct. It only needs to be set to false once after a manual refresh which occurs in the first timeout.